latch/ldata/_transfer/upload.py

```diff
@@ -48,32 +46,42 @@ class UploadJob:
     dest: str
 
 
+@dataclass(frozen=True)
+class _UploadResult:
```

Comment:
```suggestion
class UploadResult:
```

this is already in a private module